src/commands/migrate.ts

```diff
@@ -37,6 +38,11 @@ export async function run(ctx: CommandContext<typeof meta.args>) {
       .map((rep) => rep.from)
   );
 
+  // If --all flag is used, add all available migrations
+  if (all) {
+    targetModules.push(...fixableReplacementsTargets);
```

Comment:
just in case this gets super long, can you do a `for...of` instead?

a `push` with a huge amount of args can error eventually